test/integration/setup.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Global Setup & Teardown functions for the integration tests
+ */
+var path = require('path');
```

Comment:
yeah, now I am asking the same question to myself ;). I think I will revert it back to `import`